app/pb/account.proto

```diff
@@ -77,3 +98,49 @@ message GetContributorFormInfoRes {
 message MarkContributorFormFilledReq {
   bool filled_contributor_form = 1;
 }
+
+message ChangePhoneReq {
+  // New phone number in E.164 format, or empty string to remove phone number.
+  // The phone number will be visible to moderators in your area, verified or not.
```

Comment:
Do you mean site admins? Moderators sounds like community moderators, which would be a bad idea

Reply:
Reading https://community.couchers.org/t/verification-process/480/18 it sounds like the phone number should be shown to the accepted hosts and accepted guests. I will change the description to that, but don't know how the API should look like yet. Probably a new API call to get all verified data.

Reply:
Let's add that such API later!